src/math/p5.Vector.js

```diff
@@ -3901,6 +3901,22 @@ p5.Vector = class {
  * @method clampToZero
  * @return {p5.Vector} with components very close to zero replaced with zero.
  * @chainable
+ * @example
+ * <div">
+ * <code>
+ * function setup() {
+ *   // Create a new vector
+ *   let v = createVector(0.0000000000000002220446049250313, 5 );
```

Comment:
why are we using such a detailed number can't we use 0.000....02 

do we need all those digits ?

Reply:
same value is used in tests as well, 

it is ok if this value is kept, you can make that decision